site/apache.md

```diff
@@ -393,6 +391,16 @@ domain1-apache-webtier                                    2h
 ```
 
 
+It is sometimes, but rarely, desirable to expose a WebLogic Administration Server host and port through a load balancer to a public network.  If this is needed, you can customize exposure of the WebLogic Administration Server host and port by `loadBalancerExposeAdminPort property` in the `create-weblogic-domain-inputs.yaml` file.
```

Comment:
Change, "you can customize exposure of the WebLogic Administration Server host and port by `loadBalancerExposeAdminPort property` in the `create-weblogic-domain-inputs.yaml` file." 
To "you can customize the exposure of the WebLogic Administration Server host and port by using the `loadBalancerExposeAdminPort property` in the `create-weblogic-domain-inputs.yaml` file.